src/utils.ts

```diff
@@ -10,6 +10,14 @@ module.exports = {
   formatForDom: word => {
     return word.replace(/[!"#$%&'()*+,./:;<=>?@[\]^`{|}~ ]/g, '-')
   },
+  sortObject(object) {
+    return Object.keys(object)
+      .sort()
+      .reduce(function(result, key) {
+        result[key] = object[key]
+        return result
+      }, {})
```

Comment:
```suggestion
      .reduce((result, key) => {
        result[key] = object[key]
        return result
      }, {})
```

Prefer use arrow function instead